### PR TITLE
MH-12746, Library Update

### PR DESCRIPTION
**Pull request by lkiesow**
This patch updates several Java dependencies.

### CODE REVIEW

File: assemblies/karaf-features/src/main/feature/feature.xml [hunk 1]
```diff
@@ -42,23 +42,22 @@
     <bundle>mvn:com.entwinemedia.common/functional/${functional.version}</bundle>
     <bundle>mvn:com.fasterxml.jackson.core/jackson-core/${jackson.version}</bundle>
     <bundle>mvn:com.google.guava/guava/18.0</bundle>
-    <bundle>mvn:com.google.code.gson/gson/2.8.1</bundle>
-    <bundle>mvn:commons-collections/commons-collections/3.2.2</bundle>
+    <bundle>mvn:com.google.code.gson/gson/2.8.2</bundle>
+	 <bundle>mvn:commons-collections/commons-collections/${commons-collection.version}</bundle>
```
Comment (staubesv) [inline assemblies/karaf-features/src/main/feature/feature.xml]: Indentation

**Review by staubesv — CHANGES_REQUESTED** [review]
Found a cosmetic issue (indentation, see comment).

Except that, seems to work.

**Review by gregorydlogan — APPROVED** [review]

**Comment by gregorydlogan** [comment]
Approved pending fix identified by @staubesv

**Comment by lkiesow** [comment]
Indentation fixed.

**Review by staubesv — APPROVED** [review]
Works as expected.

**Pull request merged**